pkg/kubectl/resource/visitor.go

```diff
@@ -222,7 +222,7 @@ func (v *PathVisitor) Visit(fn VisitorFunc) error {
 		if !v.IgnoreErrors {
 			return err
 		}
-		fmt.Fprintf(os.Stderr, "error: unable to load file %q: %v\n", v.Path, err)
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
Which issue do you discuss more consistent error handling in kubectl. I would appreciate it if you could let me to join to do some work on that. Thanks!